<?xml version="1.0" encoding="UTF-8"?>
<com.ibm.datatools.etl.errmsgs:Message xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:com.ibm.datatools.etl.errmsgs="http:///com/ibm/datatools/etl/errmsgs.ecore" msgNum="12208" timestamp="1487302486531" msgTxt="Validation succeeded for the 'CHK'." msgSeverity="LOG" component="dataflow.validation" msgDescTxt="Validation succeeded for CHK." msgCause="" msgResponse="" entityType="DataFlow">
  <expandedMsgs msgNum="12208" timestamp="1487302486531" msgTxt="Validation succeeded for the 'flowLabel'." msgSeverity="LOG" component="dataflow.validation" msgDescTxt="Validation succeeded for flowLabel." msgCause="" msgResponse="" entityType="DataFlow" operatorID="01"/>
  <expandedMsgs msgNum="12208" timestamp="1487302486531" msgTxt="Validation succeeded for the 'flowLabel'." msgSeverity="LOG" component="dataflow.validation" msgDescTxt="Validation succeeded for flowLabel." msgCause="" msgResponse="" entityType="DataFlow" operatorID="02"/>
</com.ibm.datatools.etl.errmsgs:Message>
